usr/tgtd.h

```diff
@@ -200,6 +200,7 @@ struct scsi_lu {
 	uint64_t size;
 	uint64_t lun;
 	char *path;
+	char *vmdkid; /* Each LUN is associated with VMDK */
```

Comment:
Do not modify tgt's original DS.

Reply:
i think we will need this, since in bs_init() we initialize bs_hyc_info structure, which needs this vmdkid and to this function this scsi_lu structure is passed, which is per lun